crates/script/src/receipts.rs

```diff
@@ -38,13 +38,24 @@ pub async fn check_tx_status(
             return Ok(receipt.into());
         }
 
-        // If the tx is present in the mempool, run the pending tx future, and
-        // assume the next drop is really really real
-        Ok(PendingTransactionBuilder::new(provider, hash)
-            .with_timeout(Some(Duration::from_secs(180)))
-            .get_receipt()
-            .await
-            .map_or(TxStatus::Dropped, |r| r.into()))
+        loop {
+            if let Ok(receipt) = PendingTransactionBuilder::new(provider, hash)
+                .with_timeout(Some(Duration::from_secs(120)))
```

Comment:
180 again?

Reply:
I think this is actually the better solution because we now check if the tx still exists and don't necessarily have to increase the timeout here